nio_cli/commands/config.py

```diff
@@ -92,6 +103,28 @@ def config_ssl(name, conf_location):
     os.remove(conf_location)
     os.rename(tmp.name, conf_location)
 
+    success_message(name, True, new_certs.lower() == 'y')
+
+def success_message(name, ssl=False, self_signed=False):
+
+    print('')
+    print('\033[92m' + 'Success!' + '\033[0m')
+    print('First, start your instance by entering your project directory: ' + '\033[94m' + 'cd ' + name + '\033[0m' + ', and starting the nio daemon: '  + '\033[94m' + 'niod' + '\033[0m')
+
+    if (self_signed == True):
+        print('Next, accept your self-signed certificate by visiting ' + '\033[94m' + 'https://localhost:8181' + '\033[0m' + ' and clicking "Advanced > Proceed to Site Anyway".' + '\033[0m')
```

Comment:
Hmm, I like what you're thinking here to get the cert "validated" but I don't know if we should hardcode the localhost and 8181 parts. This might be better suited for the docs and not the CLI

Reply:
I've been thinking that we should let them choose a port (8181 = default)... because as it is right now, the CLI works exactly once if they keep the first one running.

Reply:
now we let them choose host and port